lib/api.js

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 import * as USER from "./user";
 
-const API = axios.create({
+export const API = axios.create({
```

Comment:
```suggestion
const API = axios.create({
```

Don't you need to do at the end:

```js
export default API;
```

?

Reply:
I don't think so